source/jormungandr/jormungandr/pt_planners/common.py

```diff
@@ -115,3 +115,57 @@ def send_and_receive(self, *args, **kwargs):
             return self.breaker.call(self._send_and_receive, *args, **kwargs)
         except pybreaker.CircuitBreakerError:
             raise DeadSocketException(self.name, self.zmq_socket)
+
+
+def get_crow_fly(
+    pt_planner,
+    origin,
+    streetnetwork_mode,
+    max_duration,
+    max_nb_crowfly,
+    object_type=type_pb2.STOP_POINT,
+    filter=None,
+    stop_points_nearby_duration=300,
+    request_id=None,
+    depth=2,
+    forbidden_uris=[],
+    allowed_id=[],
+    **kwargs
+):
+    logger = logging.getLogger(__name__)
+    # Getting stop_points or stop_areas using crow fly
+    # the distance of crow fly is defined by the mode speed and max_duration
+    req = request_pb2.Request()
+    req.requested_api = type_pb2.places_nearby
+    req.places_nearby.uri = origin
+    req.places_nearby.distance = kwargs.get(streetnetwork_mode, kwargs.get("walking")) * max_duration
+    req.places_nearby.depth = depth
+    req.places_nearby.count = max_nb_crowfly
+    req.places_nearby.start_page = 0
+    req.disable_feedpublisher = True
+    req.places_nearby.types.append(object_type)
+
+    allowed_id_filter = ''
+    if allowed_id is not None:
+        allowed_id_count = len(allowed_id)
+        if allowed_id_count > 0:
+            poi_ids = ('poi.id={}'.format(uri) for uri in allowed_id)
```

Comment:
these filter will no work with loki, right ?

Reply:
yep, loki cannot handle that kind of request